plugins/restful/RestfulDataProviderEFQ.php

```diff
@@ -47,6 +47,18 @@ public function getEntityType() {
   }
 
   /**
+   * Get the entity info for the current entity the endpoint handling.
+   *
+   * @param null $type
+   *   The entity type. Optional.
+   * @return array
+   *   The entity info.
+   */
+  public function getEntityInfo($type = NULL) {
+    return entity_get_info($type ? $type : $this->getEntityType());
```

Comment:
`return entity_get_info($type ?: $this->getEntityType());` won't work in php 5.3